app/src/main/java/co/develhope/meteoapp/data/Data.kt

```diff
@@ -12,6 +13,18 @@ import org.threeten.bp.OffsetDateTime
 
 
 object Data {
+
+    fun getSearchData() : List<DataSearches>{
+        return  listOf<DataSearches>(
```

Comment:
Non hai tenuto la separazione tra oggetto del dominio ( il dato in sè) con l'oggetto che rappresenta la ui